script.js

```diff
@@ -0,0 +1,6 @@
+const iconMenu = document.querySelector('#ml');
+const menu = document.getElementById('it-nav');
+iconMenu.addEventListener('click', () => {
+  menu.classList.toggle('showMenu');
+  iconMenu.classList.toggle('changeIcon');
+});
```

Comment:
- [ ] As it is a [requirement ](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md) for this project, "`When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.`", please make sure the mobile menu disappears when a user selects of tap on a particular option. After selecting an option, the mobile menu doesn't disappear, please kindly fix that.

Reply:
As the previous reviewer requested, please make sure after clicking on any link, the nav closes.

![Screen Recording 2023-03-08 at 1 32 19 AM](https://user-images.githubusercontent.com/25887253/223569013-efa485be-62bb-473a-a0b1-855311a953e4.gif)